### ISSUE TITLE
Not enough vertical alignment

### ISSUE THREAD

**Issue by spring378124** [kupiqu/plasma-active-application#11]
The height of the active menu is different from the height of the global menu text.
![screenshot_20181111_192052](https://user-images.githubusercontent.com/44946494/48314676-9d5e9a00-e5ee-11e8-8d93-fc2c42cce22b.png)

**Comment by kupiqu** [comment]
Yes, by a tiny bit. I guess labels and buttons are not aligned the same way :(
